### PR TITLE
Enable Deck Rerun button

### PR DESCRIPTION
**Pull request by JoelSpeed**
This enables certain users to rerun jobs from the UI should they fail

### CODE REVIEW

File: prow/sealedsecret-cookie-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: cookie-secret
+  namespace: deafult
+spec:
+  encryptedData:
+    secret: AgBaXrbLd5NV0HXZxUDfOxy200HKK+2E3tTr8/SjfjOAlHPh5NC9ohxycffv5fP1pj8srpJRmAxSgtSYVz5/oyfdKhC2re0pnwKKEErsAlyYSAcub7JNcuB7MVCRoigRx9CN5/pIEZSWIPEJLVCa9cTT+FtA4P2cYfTzaCEVKfvQXY2i0N0XthO1UNORxPcjtCYnAebNan+SBcdA4p1xHINjgRKlMr/7pzxdRqxPKZl6K+0lRYPOia8wQo9ez3p1jrYzXYbOmY+MJV8bRlRIqbRs8vvug9lhytVcHZU7h03OLSUOxUb5/4BdUa5oUJA+sEdE6Ozxi2ZOnzvkWewq9UVY4qIrKsYlAvzadVoT312/ZXprWMz6+iGF2EX5ygZbVxC07D96dy/y55Hnr2GvdPfj3elq9VCeM+AbwtZ7VpC7r3Bg3fdBiQUiiQsORmPvasAYSBy4wrYGKoq6x+1fNX5XeLP4WK+naKkB3egR6Y9+HhPywVNKu1d92rtNfj4t5Vh1P/LAxoxXyKdriDh34Teg9Y3sy3ZNBt3kWvaedjytUfoT+S373VR3iFad1M4lO6ObFPpKwg4EqXKIP/HoysFEhqo+CwDgdhQwDTD6bRvHa38a2RUGn/TCUSruY4tOvkQCRJogr64g0TmvG8m9xGCfYK1SBtME++HXB5lckFg8KqwklvHGrB9DwgR8rNAjsJVNlxF9UJ2UtszE+v1zgYNyRybp5yq8aAePOtXJY7qD1eBkciPOgZZfVA4o5A==
+
```
Comment (mthssdrbrg) [inline prow/sealedsecret-cookie-secret.yaml]: Nitpick; remove empty newline and unnecessary `metadata.creationTimestamp`.

File: prow/sealedsecret-github-oauth-config.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: github-oauth-config
+  namespace: default
+spec:
+  encryptedData:
+    secret: AgBQIRLxwRPxAJ8NDFSAPn78Dr9Ca58YKzsnR73bJGDVO5KIWmVcOunRoeJy/T43fYOJXbny+tbbo9dsUeLgbXLZx76ASOcqy8Quyif0yY2M2jqV3yVvwESKa0BV/Ygdt0C+69QrFDwENUat2T+71JtVyLn3ka+/CuIpIxcSwFijnn3vsXORzXWysDnTOTqWU/sbXyexWvFlkaKX0dDQG+pwrXrk5Fytf6wZoqhdEthzFxYw16pDcuD9N3meTS4z8YMQ8RUb0LoStpB+0yoA15m1vzlYB39mpak8s9mKskEjyld0BtPfW+vVwexuoL26W47uRbpKK6NU9dorXwRo4GBToQyXyPUCtAsKY4MasalRgZm7Wn+Gk5qFIhRlfhI+qECpOMx92NKed03PmyB8MQuyUYHKMBtxsCo1dV07/LEdsUli31PfWKrUiBFEE1oA2HJJFuTvWbjTpLejotMnpUyqQ1VGvPnA2qm3A+OD1YllU4Zs1PAJm9XW3a+7w2EBye9tXSqeH8f0TLze+7uJ6ilfSNRgYU0TB2JI7B3GyP9vU4t7iKnmQc4R7Zvc/Wyx7vtDGvgyaFLcZJ92Y9vbjt5bYCcMMohlwqHFguCCKckt6h60HWy3QQmIgIgSNzrD6idwuLwKQifdcbZsBL03bUhBFSTDYmm8Qevva1ElmLDF5sDxabNzy57f7GWUFRsNq4hC1Vt6M1Jcqxmuees1hRt1BEZbYocVvrvW8ywzSdjhafkn8f2cj2uyo22fkM23zfchYtlwBVxL++Ooa10gGMMyT/RsWPW39QWjsU5wZa30DcjjG2ktwUDiVv8VqHN0v4LvfJMN/ITN1O4udH3VP3Lyal9NdyUOVKKc+yriXrCSEMQEDUa1F6nOHmk0cLO8g1cqLUI+
+
```
Comment (mthssdrbrg) [inline prow/sealedsecret-github-oauth-config.yaml]: Same as above.

**Review by mthssdrbrg — COMMENTED** [review]
Looks good. Just two minor nitpicks :see_no_evil:

**Comment by JoelSpeed** [comment]
@mthssdrbrg Have tested this and fixed up your comments, should be good to go

**Review by mthssdrbrg — APPROVED** [review]

**Pull request merged**